examples/batched-processing/Makefile

```diff
@@ -0,0 +1,9 @@
+build:
+	docker build -t prio-dev ../..
+	docker-compose build
+
+up:
+	docker-compose up
```

Comment:
Should we just call this target `test`?

Kind of weird that this just sort of hangs around doing nothing after prio completes. Can we just make it do that?

Reply:
I think the container is designed to be long-living, since it is an object store. I don't think there's a way to automatically clean up. I've been manually killing the process and then running `make clean`.

Reply:
Well, it's basically just a test command, so I'm not sure if there's much value in keeping it around?

Reply:
It is useful for inspecting the results afterwards, but in this testing context you're right. It might be really hacky, as opposed to just calling `make clean` once the result appears. I had the same problem in the `examples/docker-asyncio` project.

Reply:
I've added to the documentation the preferred method of running this. I think getting this into CI would be a bit tricker since it would require some docker-in-docker in docker trickery.

```bash
make build
make test &
# when done
make clean
```

Reply:
Running docker-compose in ci is not difficult, here's an example of us doing it for iodide:

https://github.com/iodide-project/iodide/blob/c7dcf7b76c1b3cb50ffba583bea75ed3124d8e37/.circleci/config.yml#L121

I'd encourage you to give it a try! Let me know if I'm missing something.

Reply:
Also here's how the tests get executed: https://github.com/iodide-project/iodide/blob/c7dcf7b76c1b3cb50ffba583bea75ed3124d8e37/Makefile#L27